src/test/java/com/cosmian/TestCoverCrypt.java

```diff
@@ -143,10 +143,11 @@ public void test_keys_import_export() throws Exception {
     @Test
     public void test_user_decryption_keys() throws Exception {
 
-        if (!TestUtils.serverAvailable(TestUtils.kmsServerUrl())) {
-            System.out.println("No KMS Server: ignoring");
+        if (TestUtils.isGithub()) {
```

Comment:
Does this cover running tests locally when you do not have a KMS Server available ?

Reply:
Tests will be run only on Gitlab CI with a KMS server up